demo/full/bxf-parser2/src/parser/schedule/index.js

```diff
@@ -0,0 +1,95 @@
+import reduceChildren from "./../reduce_children";
+import {
+  parseEventData,
+  parseContentData,
+} from "./schedule_elements";
+
+/**
+ * Parse entire schedule
+ * @param {Element} node
+ */
+export default function parseSchedule(node) {
+  return reduceChildren(node, (res, name, node) => {
+    switch (name.toLowerCase()) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: "switch" statements should have at least 3 "case" clauses ([javascript:S1301](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS1301))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=366&issues=AWbt7Z3qW3Y69bz5aGXC&open=AWbt7Z3qW3Y69bz5aGXC)

[ISSUE_KEY]: AWbt7Z3qW3Y69bz5aGXC